feature/gnmi/ate_tests/telemetry_interface_packet_counters_test/telemetry_interface_packet_counters_test.go

```diff
@@ -341,14 +341,17 @@ func ConfigureDUTIntf(t *testing.T, dut *ondatra.DUTDevice) {
 		// per: https://github.com/openconfig/featureprofiles/issues/253
 		i.Enabled = ygot.Bool(true)
 		s.Enabled = ygot.Bool(true)
-		v4.Enabled = ygot.Bool(true)
+		if !*deviations.IPV4Enabled {
+			v4.Enabled = ygot.Bool(true)
+		}
 		v6.Enabled = ygot.Bool(true)
 
 		dut.Config().Interface(intf.intfName).Replace(t, i)
 
 		t.Logf("Validate that IPv4 and IPv6 addresses are enabled: %s", intf.intfName)
 		subint := dut.Telemetry().Interface(intf.intfName).Subinterface(0)
-		if !subint.Ipv4().Enabled().Get(t) {
+
+		if !*deviations.IPV4Enabled && !subint.Ipv4().Enabled().Get(t) {
 			t.Errorf("Ipv4().Enabled().Get(t) for interface %v: got false, want true", intf.intfName)
 		}
```

Comment:
Probably easier to read:

```suggestion
		if !*deviations.IPv4MissingEnabled {
			if !subint.Ipv4().Enabled().Get(t) {
				t.Errorf("Ipv4().Enabled().Get(t) for interface %v: got false, want true", intf.intfName)
			}
		}
```

Reply:
thanks, addressed